src/Console/Command/ScopeHintCommand.php

```diff
@@ -61,7 +61,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         } else {
             $table = new Table($output);
             $table
-                ->setHeaders(['website', 'store', 'values'])
+                ->setHeaders(['scope', 'scope_id', 'path', 'values'])
```

Comment:
Shouldn't this be `scope_code` instead of `scope_id`? As a user, I'd expect a numeric ID in that column.

Reply:
@avstudnitz In the ticket https://redmine.integer-net.de/issues/26061 that @schmengler created it says scope_id.
@schmengler Should I change it to scope_code?

Reply:
please do it in the same way as in `core_config_data`

Reply:
@schmengler I checked the `core_config_data`. There it says `scope_id`. So I will not change the name.